server/models/articleVotes.js

```diff
@@ -0,0 +1,44 @@
+export default (sequelize, DataTypes) => {
+  const ArticleVote = sequelize.define(
+    'ArticleVote',
+    {
+      id: {
+        type: DataTypes.UUID,
+        primaryKey: true,
+        allowNull: false,
+        defaultValue: DataTypes.UUIDV4
+      },
+      vote: {
+        type: DataTypes.INTEGER,
```

Comment:
Why are we not considering ENUM instead integer?

Reply:
After having a few headaches trying to set a default value for the enum and not succeeding. I moved back to an integer and added a middleware that validates the `vote` input